core/capabilities/launcher.go

```diff
@@ -118,6 +118,99 @@ func NewLauncher(
 	}
 }
 
+// CRE-326: Maintain only necessary Don2Don connections:
```

Comment:
nit: Prefer not to reference project management tickets.

These lines seem like they would be better fit below per method doc.